www/%username/widgets/index.html.spt

```diff
@@ -68,5 +68,25 @@ response.headers[b'Content-Security-Policy'] = csp
     <p><script src="giving.js"></script></p>
     % endif
 
+    <h3>{{ _("Shields.io Badges") }}</h3>
```

Comment:
It might not be immediately clear to translators and users that "Shields.io" is a domain name here. Maybe "Badges (shields.io)" or "Badges from shields.io" would be better?